libs/vertexai/langchain_google_vertexai/_utils.py

```diff
@@ -113,9 +111,9 @@ def get_client_info(module: Optional[str] = None) -> "ClientInfo":
     Returns:
         google.api_core.gapic_v1.client_info.ClientInfo
     """
-    client_library_version, user_agent = get_user_agent(module)
+    gapic_version, user_agent = get_user_agent(module)
     return ClientInfo(
-        client_library_version=client_library_version,
```

Comment:
let's keep both, please

we might rely on client_library_version in some other place

Reply:
just send both client_library_version and additionally gapic_version= client_library_version